modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -86,54 +86,54 @@ EOF
 $ oc get csv -n openshift-tempo-operator
 ----
 
-. Create a secret for your object storage bucket by running one of the following commands:
+. Create a project of your choice for the *TempoStack* instance that you will create in a subsequent step:
 
-** To create a secret from a YAML file:
+** To create a project from standard input without metadata:
 +
 [source,terminal]
 ----
-$ oc apply -f <secret_file>.yaml
+$ oc new-project <project_of_tempostack_instance>
 ----
 
-** To create a secret from standard input:
+** To create a project from standard input with metadata:
 +
 [source,terminal]
 ----
 $ oc apply -f - << EOF
-<object_storage_secret>
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: <project_of_tempostack_instance>
 EOF
 ----
-+
---
-include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
---
-+
---
-include::snippets/distr-tracing-tempo-secret-example.adoc[]
---
 
-. Create a project of your choice for the *TempoStack* instance that you will create in the next step:
+. In the project that you created for the *TempoStack* instance, create a secret for your object storage bucket by running one of the following commands:
 
-** To create a project from standard input without metadata:
+** To create a secret from a YAML file:
 +
 [source,terminal]
 ----
-$ oc new-project <project_of_tempostack_instance>
+$ oc apply -f <secret_file>.yaml
 ----
 
-** To create a project from standard input with metadata:
+** To create a secret from standard input:
 +
 [source,terminal]
 ----
 $ oc apply -f - << EOF
-apiVersion: project.openshift.io/v1
-kind: Project
-metadata:
-  name: <project_of_tempostack_instance>
+<object_storage_secret>
 EOF
 ----
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
```

Comment:
Again, I realize this was already here, but this seems like these tables would be better put into a reference module, rather than interrupting the flow of the procedure for it.

Reply:
Thank you for your suggestion. 😃 Wouldn't having the reader look up the reference module interrupt the flow of the procedure even more?